README.md

```diff
@@ -1,2 +1,200 @@
+<img align="right" src="https://github.com/moderepo/sensor-cloud-eval/blob/master/sceval_frontend/src/common_images/mode-logo.png">
+
 # Sensor-Cloud-Eval
-A TSX version of Sensor Cloud to be used as an evaluation kit by partners.
+
+Sensor Cloud Eval is a **TSX version** of MODE's Sensor Cloud, developed to be used as an evaluation kit by business partners and app developers.
+
+## Contents
+
+- [Features](#features)
+- [Installation and Setup](#installation-and-setup)
+- [Prerequisites](#prerequisites)
+- [Quick start](#quick-start)
+- [Mode API Reference](https://dev.tinkermode.com/docs/api/)
+- [API Examples](#api-examples)
+  - [Using REST Endpoints (GET, POST, PUT, PATCH, DELETE)](#using-rest-endpoints)
+    - [Handling User Information](#handling-user-information)
+    - [Linking / Unlinking Sensor Modules](#linking-and-unlinking-sensor-modules)
+    - [Sending Device Commands](#sending-device-commands)
+  - [Receiving Device Events (Websocket)](#receiving-device-events)
+- [Links](#links)
+
+## Features
+
+<img src="https://github.com/moderepo/sensor-cloud-eval/blob/app/readme/sceval_frontend/src/common_images/screenshots/sensor_module.png" alt="Size Limit CLI" width="738">
+
+- Management of sensor modules and their device associations from within the application. :electric_plug:
+
+- Ability to view real-time sensor data in multiple sensor modules across multiple devices. :chart_with_upwards_trend:
+
+- Ability to view time-series sensor data and toggle between various time-horizons as well as sensing intervals. :clock10:
+
+- Configuration of sensor module sensing and general settings. :arrows_counterclockwise:
+
+## Installation and Setup
+
+To begin development off MODE's sensor cloud evaluation kit, first clone the repository to your local machine.
+```sh
+$ git clone https://github.com/moderepo/sensor-cloud-eval.git
+```
+
+## Prerequisites
+
+1. You will need MODE, Inc. to pre-provision the gateway(s) to your project before running the application
+
+2. You will also need to have any sensors you wish to pair with your gateway(s) in your possession.
+
+3. You must have an account with the [Tinkermode Console](https://console.tinkermode.com/console/signup).
+
+4. If you don't already have npm and typescript installed, you will need to install both in order to run the application.
+
+```sh
+$ npm install -g npm
+$ npm install -g typescript
+```
+
+## Quick Start
+
+**Assuming you have met the previously mentioned prerequisites**, follow these steps to get up and running in minutes:
+
+1. `cd` into `sceval_frontend` and run `npm i` or `npm install`. This will install the default app dependencies.
+
+2. Run `npm start` to immediately bring up the application at http://localhost:3000.
+
+3. Navigate to http://localhost:3000 your browser to view the running application.
+
```

Comment:
I think it's better explaining how to use this application like the way of adding sensor.
It's good writing it on this section, or it's also good we use GitHub's wiki for this.

Reply:
Okay, I will go through all steps in addition to listing device commands.